rook.md

```diff
@@ -0,0 +1,496 @@
+existing install_yamls target.
+
+**Note:**
+The rest of the guide assumes that you're working in a network isolation
+context and `Ceph` should be deployed using the `EDPM` node storage network,
+propagated to the ctlplane via nncp / net-attach-def.
+
+```
+make edpm_deploy
+```
+
+### Deploy Ceph
+
+In a network-isolated context, verify the `EDPM` node is deployed and `AEE`
+completed the execution:
+
+```bash
+NAME                                                          READY   STATUS      RESTARTS   AGE
+bootstrap-edpm-deployment-openstack-edpm-ipam-hbhnh           0/1     Completed   0          9h
+configure-network-edpm-deployment-openstack-edpm-ipam-6gzwq   0/1     Completed   0          9h
+configure-os-edpm-deployment-openstack-edpm-ipam-bdzwx        0/1     Completed   0          9h
+download-cache-edpm-deployment-openstack-edpm-ipam-bbqll      0/1     Completed   0          9h
+install-certs-edpm-deployment-openstack-edpm-ipam-cnzgf       0/1     Completed   0          9h
+install-os-edpm-deployment-openstack-edpm-ipam-bfgl4          0/1     Completed   0          9h
+libvirt-edpm-deployment-openstack-edpm-ipam-98zlt             0/1     Completed   0          9h
+neutron-metadata-edpm-deployment-openstack-edpm-ipam-xt8gg    0/1     Completed   0          9h
+nova-edpm-deployment-openstack-edpm-ipam-kprc6                0/1     Completed   0          8h
+ovn-edpm-deployment-openstack-edpm-ipam-mpv7q                 0/1     Completed   0          9h
+reboot-os-edpm-deployment-openstack-edpm-ipam-mjhmw           0/1     Completed   0          9h
+repo-setup-edpm-deployment-openstack-edpm-ipam-q5k55          0/1     Completed   0          9h
+run-os-edpm-deployment-openstack-edpm-ipam-t4q9p              0/1     Completed   0          9h
+ssh-known-hosts-edpm-deployment-j5gqk                         0/1     Completed   0          9h
+telemetry-edpm-deployment-openstack-edpm-ipam-bhz7j           0/1     Completed   0          8h
+validate-network-edpm-deployment-openstack-edpm-ipam-t7rh9    0/1     Completed   0          9h
+```
+
+1. Ensure the `vlan21` interface (e.g., 172.18.0.100) is on the storage
+   network.
+
+```
+$ ip -o -4 a
+
+1: lo    inet 127.0.0.1/8 scope host lo\       valid_lft forever preferred_lft forever
+4: br-ex    inet 192.168.122.100/24 brd 192.168.122.255 scope global br-ex\       valid_lft forever preferred_lft forever
+5: vlan20    inet 172.17.0.100/24 brd 172.17.0.255 scope global vlan20\       valid_lft forever preferred_lft forever
+6: vlan21    inet 172.18.0.100/24 brd 172.18.0.255 scope global vlan21\       valid_lft forever preferred_lft forever
+7: vlan22    inet 172.19.0.100/24 brd 172.19.0.255 scope global vlan22\       valid_lft forever preferred_lft forever
+11: podman0    inet 10.255.255.1/24 brd 10.255.255.255 scope global podman0\       valid_lft forever preferred_lft forever
+```
+
+2. Clone the Ceph deployment scripts:
+
+```bash
+git clone https://github.com/fmount/openstack-ceph
```

Comment:
should we consider adding those scripts to a directory in install_yamls? or we add them in the devsetup folder and `make edpm_compute` already adds them to the image used to create the edpm nodes? could probably also automate there the `Install dependencies on the EDPM node` part. just an idea for discussion which we can do in a follow up

Reply:
Yeah, this is definitely something we can discuss. Right now we don't make assumptions on the ceph deploy, but I'm ok to move those scripts in install_yamls if that helps (and update this patch when the move is done).

Reply:
@stuggi FYI I can update this section once [1] lands and do not require any external dependency.

[1] https://github.com/openstack-k8s-operators/install_yamls/pull/929

Reply:
I actually updated the section to not require any external dependency, and point to `devsetup/ceph` for any ceph related script.

Reply:
Scripts are merged in install_yamls now,